pipeline/outputs/observe.md

```diff
@@ -29,18 +28,18 @@ In your main configuration file, append the following _Input_ & _Output_ section
 [OUTPUT]
     name         http
     match        *
-    host         collect.observeinc.com
+    host         my-observe-customer-id.collect.observeinc.com
     port         443
     tls          on
 
-    # For Windows: provide path to root cert
-    #tls.ca_file  C:\td-agent-bit\isrgrootx1.pem
-
-    http_user    ${OBSERVE_CUSTOMER}
-    http_passwd  ${OBSERVE_TOKEN}
     uri          /v1/http/fluentbit
 
     format       msgpack
+    header       Authorization     Bearer ${OBSERVE_TOKEN}
     header       X-Observe-Decoder fluent
     compress     gzip
+
+    # For Windows: provide path to root cert
+    #tls.ca_file  C:\td-agent-bit\isrgrootx1.pem
```

Comment:
Ah, not sure if this was just added but `td-agent-bit` is deprecated and removed from 1.9.9 so could you switch it to `fluent-bit`?

Reply:
Done.